### ISSUE TITLE
Is configuring the plaintext extension possible?

### ISSUE THREAD

**Issue by john-griffin** [maizzle/framework#384]
Could not find it in the docs, is it possible to change the plaintext output file's extension? It appears to be hardcoded to `txt`. I've been able to change the html extension without a problem.

**Comment by cossssmin** [comment]
Right now this is hardcoded indeed, but we could make it configurable, sure.

I imagine we could do it like with other transformers: 

- just setting `plaintext: true` would generate `.txt` files at the same location as the HTML version (current behavior)
- using an object would allow for configuring extension and maybe even destination, in case you need the plaintext version output somewhere else on your filesystem:

    ```js
    plaintext: {
      enabled: true,
      extension: 'txt', // string, excluding the dot
      path: 'some/other/directory', // string | path to output file at (maybe name it 'destination'?)
    }
    ```

    In a template, you would use that like this with Front Matter:

    ```
    ---
    plaintext:
      enabled: true
      extension: txt
      path: some/other/directory
    ---
    ```

I'll add this to the backlog and start working on a draft PR, thanks for bringing it up 🙂

**Comment by john-griffin** [comment]
Thanks, yeah that makes sense. Maybe the config could be closer to the html config for consistency? Something like:

``` js
plaintext: {
  enabled: true,
  destination: {
    path: 'some/other/directory',
    extension: 'text',
  }
}
```

**Comment by cossssmin** [comment]
Yup good call 👍

**Comment by cossssmin** [comment]
Available in [v3.1.0](https://github.com/maizzle/framework/releases/tag/v3.1.0) 👍
